e2e/src/comics.e2e-spec.ts

```diff
@@ -30,6 +30,18 @@ describe('Comics', () => {
     );
   });
 
+  it(
```

Comment:
When there is only one `it` statement I would keep that inline, e.g.

```suggestion
  it('should display the selected comic cover when zoomed', async () => {
```

but up to you